.github/workflows/release.yml

```diff
@@ -46,3 +46,18 @@ jobs:
         run: |
           make buildx.image
           make buildx.push
+
+  trivy:
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+        
+      - name: Run Trivy vulnerability scanner
+        uses: aquasecurity/trivy-action@master
+        with:
+          image-ref: 'openebs/linux-utils:ci'
```

Comment:
In this file we need to use [this](https://github.com/openebs/linux-utils/pull/8/files#diff-87db21a973eed4fef5f32b267aa60fcee5cbdf03c67fafdc2a9b553bb0b15f34R30-R31) tag. Since the release.yml file produces a tagged image.

Reply:
Which tag ? link you have given redirects to same page.

Reply:
Oops, sorry. Can you check [this](https://github.com/openebs/linux-utils/blob/ef4802a3d87605a8fb2cd038b0edbb2fe5e75cbf/.github/workflows/release.yml#L30). This will be the tag. You need to use that tag here.

Reply:
I have added release tag. I hope [this](https://github.com/prakhargurunani/linux-utils/blob/0da64faf35d6a05c0fdd934f4b8ca691a64fdfbc/.github/workflows/release.yml#L59) is what we wanted.

Reply:
Not like this, this will give us only, for example, `2.2.0`, what we need is `openebs/linux-utils:2.2.0`